config_parse.go

```diff
@@ -8,18 +8,34 @@ import (
 //DEBUG flag toggles log output
 var DEBUG bool
 var config Config
+var authConfig Auth
+
+//DefaultAttachmentDoc is used as a template
+//for attachment documents. If no attachment doc exists,
+//this will be used instead
+var DefaultAttachmentDoc string
 
 //Config represents all resource and sync endpoints
 type Config struct {
 	ResourcesDir string `json:"resourcesDir"`
 	SyncURL      string `json:"syncUrl"`
 	Bucket       string `json:"bucket"`
+	Auth         Auth   `json:"auth"`
 }
 
 //ConfigJSON represents global config properties
 type ConfigJSON struct {
```

Comment:
I like the approach where structs, functions and interfaces are grouped in files by feature.